spring-cloud-gcp-docs/src/main/asciidoc/spring-stream.adoc

```diff
@@ -58,10 +58,12 @@ spring.cloud.stream.gcp.pubsub.bindings.events.producer.auto-create-resources=tr
 
 If automatic resource creation is turned ON and the subscription and/or the topic do not exist for a consumer, a subscription and potentially a topic will be created.
 The topic name will be the same as the destination name, and the subscription name will be the destination name followed by the consumer group name.
-If the consumer group is not specified, an anonymous one will be created with the name `anonymous.<destinationName>.<randomUUID>`.
+
+Regardless of the automatic resource creation setting, if the consumer group is not specified, an anonymous one will be created with the name `anonymous.<destinationName>.<randomUUID>`.
```

Comment:
Why not use another randomly generated string instead of "anonymous" in the name? 

"anonymous" is a common word and might be used in a different context. (for example,  a user might have "anonymous" refer to the subject matter the program is about, like poems from known poets or anonymous ones). I don't think it should be reserved for use here.

Reply:
That's a terminology has chosen in Spring Cloud Stream: https://docs.spring.io/spring-cloud-stream/docs/current/reference/htmlsingle/#_common_binding_properties
> group

>    The consumer group of the channel. Applies only to inbound bindings. See Consumer Groups.

>    Default: null (indicating an anonymous consumer).

That's not our own call here.

If you have some concern on the matter, feel free to raise an issue against that project.